recipes/latentstrainanalysis/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "LatentStrainAnalysis" %}
+{% set version = "1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  git_rev: 4db66e9
+  git_url: https://github.com/douglasgscofield/{{ name }}.git
```

Comment:
There is no release in this repo. Downloading the `master` branch (via `https://github.com/douglasgscofield/LatentStrainAnalysis/archive/master.zip`) could fail if there are major changes on `master`. Is there any other solution?